web/source/tsconfig.embedded.json

```diff
@@ -1,24 +1,39 @@
 {
   "extends": "./tsconfig.base.json",
+
   "compilerOptions": {
 		"outFile": "../intermediate/keyman.js"
   },
-  "references": [
-    { "path": "../../resources/web-environment" },
-    { "path": "../../common/core/web/utils" },
-    { "path": "../../common/core/web/keyboard-processor/src" },
-    { "path": "../../common/core/web/input-processor/src" },
-  ],
+
   "include": [
     "./*.ts",
     "./**/*.ts"
   ],
+
   "files": [
-		"../../common/predictive-text/web-defaultWorker.ts",
     "kmwbase.ts",
 		"keymanweb.ts",
 		"kmwembedded.ts",
 		"kmwinit.ts",
 		"kmwapi.ts"
-  ]
+  ],
+
+	"exclude": [
+		"./kmwnative.ts",
+		"./kmwuibutton.ts",
+		"./kmwuifloat.ts",
+		"./kmwuitoggle.ts",
+		"./kmwuitoolbar.ts",
+	],
+
+  "references": [
+    { "path": "../../resources/web-environment", "prepend": true },
+    { "path": "../../common/core/web/utils", "prepend": true },
+    { "path": "../../common/predictive-text/browser.tsconfig.json", "prepend": true },
+    { "path": "../../common/core/web/input-processor/src", "prepend": true },
+    { "path": "../../common/core/web/keyboard-processor/src", "prepend": true },
+    { "path": "../../common/web/es6-shim", "prepend": true },
+    { "path": "../../common/web/lm-message-types" },
+    { "path": "../../common/core/web/tools/sentry-manager/src" }, // We don't prepend this, because it will be loaded separately
```

Comment:
Is this reference (to `sentry-manager`) actually even needed?  KeymanWeb isn't supposed to compile against that code.

Reply:
I just did a test build this way on my end; no issues when I deleted the line.

Reply:
```suggestion

```